slack-base/src/main/java/com/hubspot/slack/client/models/blocks/ActionsIF.java

```diff
@@ -0,0 +1,26 @@
+package com.hubspot.slack.client.models.blocks;
+
+import java.util.List;
+
+import org.immutables.value.Value;
+import org.immutables.value.Value.Immutable;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.blocks.elements.BlockElement;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ActionsIF extends Block {
+  String TYPE = "actions";
```

Comment:
Would this not be better as an enum?

Reply:
I did think about that, but there was more code involved (since we need to send it as a string), and since this isn't a value that anyone should be interacting with, I decided it was easier to leave it as a `String` in each interface.  Do you think there would be an advantage to using an enum instead?

Reply:
I just thought it would be a better fit but your reasoning makes sense